Packages/Netherlands3D/T3DPipeline/Runtime/Scripts/ParseCityJSON/CityJSONStructure/CityObject.cs

```diff
@@ -209,6 +199,28 @@ public void AddAttribute(CityObjectAttribute attribute)
             Attributes.Add(attribute);
         }
 
+        public static CityObject CreateEmpty(string id, CityObjectType type = CityObjectType.GenericCityObject, CoordinateSystem coordinateSystem = CoordinateSystem.RD)
+        {
+            var co = new GameObject().AddComponent<CityObject>();
```

Comment:
Zou de id ook de name van de gameObject kunnen zijn? misschien makkelijk als je deze Debug.Logged of in de hierarchy ziet?

Reply:
Het begon als een CityJSON lezer dus de id van de CityJSON wordt de naam van het GameObject, we zouden het later kunnen aanpassen om de id geen aparte variabele te maken maar naar gameObject.name te laten refereren